CMakeLists.txt

```diff
@@ -173,4 +173,39 @@ endif(HYDROCHRONO_ENABLE_TESTS)
 # Not a good idea to copy DLLs
 # dosen't work on Linux
 #set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR})
-#add_DLL_copy_command("${CHRONO_DLLS}")
\ No newline at end of file
+#add_DLL_copy_command("${CHRONO_DLLS}")
+
+
+# Export to include as external library in other projects
+
+install(TARGETS HydroChrono
+    EXPORT HydroChronoTargets
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+)
+
+install(
+    EXPORT HydroChronoTargets
+    FILE HydroChronoTargets.cmake
+    DESTINATION "${CMAKE_INSTALL_DATADIR}/HydroChrono/cmake"
+    NAMESPACE HydroChrono::
+)
+
```

Comment:
Missing installation of public include files
install(DIRECTORY include
    DESTINTATION ${CMAKE_INSTALL_INCLUDEDIR}
)

Reply:
Added